### ISSUE TITLE
Add a a11y audit to the styleguide tests

### ISSUE THREAD

**Issue by mansona** [ember-learn/ember-styleguide#467]
We noticed that something that uses ember-styleguide was failing an a11y check, so it's probably a good idea to make sure that our CI is running the same or similar checks 👍
